backend/routes/admin.py

```
@@ -0,0 +1,836 @@
+        return LoginResponse(
+            success=True, message="Login successful", user=user_data, session_id=auth_result["session_id"]
+        )
+
+    except Exception as e:
+        logger.error(f"Login error: {str(e)}", exc_info=True)
+        raise HTTPException(status_code=500, detail="Login failed")
+
+
+@router.post("/auth/logout")
+async def logout(
+    request: Request, response: Response, session: Dict[str, Any] = Depends(require_admin_auth)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)